pgx/src/pgbox.rs

```diff
@@ -309,10 +282,15 @@ impl<T, AllocatedBy: WhoAllocated> PgBox<T, AllocatedBy> {
     /// ## Examples
     /// ```rust,no_run
     /// use pgx::{PgBox, pg_sys};
-    /// let create_trigger_statement = PgBox::<pg_sys::CreateTrigStmt>::alloc_node(pg_sys::NodeTag_T_CreateTrigStmt);
+    /// let create_trigger_statement = unsafe { PgBox::<pg_sys::CreateTrigStmt>::alloc_node(pg_sys::NodeTag_T_CreateTrigStmt) };
     /// ```
+    ///
+    /// # Safety
+    ///
+    /// This function is unsafe as we cannot ensure that the MemoryContext used to allocate will
```

Comment:
It also returns a zeroed value (which may be invalid for T), and is probably UB if `T: pg_sys::PgNode` isn't true (can we add that bound?), since it writes the tag to the start.

Reply:
taken care of in 8bcb127.  Thanks!